modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -175,6 +176,13 @@
 				var/mob/living/carbon/C = user
 				C.head_update(src, forced = 1)
 
+/obj/item/clothing/head/helmet/sec/update_icon_state()
+	. = ..()
+	if(attached_light)
+		//This compresses it down nicely. End result is Initial(is the visor toggled)-(is the flashlight on)
+		icon_state = "[initial(icon_state)][up ? "up" : ""][attached_light.on ? "-flight-on" : "-flight"]"
+	return
```

Comment:
```suggestion
```

Reply:
oop, my bad, that was leftover from the initial testing shiz